metron-platform/metron-test-utilities/src/main/java/org/apache/metron/test/mock/MockHTable.java

```diff
@@ -70,7 +93,7 @@ public static void clear() {
   private final String tableName;
   private final List<String> columnFamilies = new ArrayList<>();
   private HColumnDescriptor[] descriptors;
-
+  private List<Put> putLog;
```

Comment:
Yeah, given that it's a test class and the amount of testing effort, I'm not really worried about it too much.  I'm good, +1.